packages/clerk-js/src/ui/components/OrganizationSwitcher/OrganizationSwitcherPopover.tsx

```diff
@@ -144,8 +149,9 @@ export const OrganizationSwitcherPopover = React.forwardRef<HTMLDivElement, Orga
                   organization={currentOrg}
                   user={user}
                   sx={theme => t => ({ padding: `0 ${theme.space.$6}`, marginBottom: t.space.$2 })}
+                  aria-hidden='true'
```

Comment:
Why do you hide this?

Reply:
menu only allows for "interactive" elements to be menu items

Reply:
aria-hidden should be used for these things:

- Purely decorative content, such as icons or images
- Duplicated content, such as repeated text
- Offscreen or collapsed content, such as menus

That's not the case for that item, thus we shouldn't hide it. You'll need to construct the menu in another way then